tests/keras_contrib/backend/test_backends.py

```diff
@@ -99,6 +100,32 @@ def test_extract(self):
                 assert zth.shape == ztf.shape
                 assert_allclose(zth, ztf, atol=1e-02)
 
+    def test_moments(self):
+        input_shape = (10, 10, 10, 10)
+        x_0 = np.zeros(input_shape)
+        x_1 = np.ones(input_shape)
+        x_random = np.random.random(input_shape)
+
+        th_axes = [0, 2, 3]
+        tf_axes = [0, 1, 2]
```

Comment:
Wait, how can that possibly be? Why should these be the same with different axes numbers? I don't see why the different ordering of axes between tf and th would apply here.

`th_mean, th_var = KCTH.moments(ip_th, 2, keep_dims=keep_dims)`
`tf_mean, tf_var = KCTF.moments(ip_tf, 1, keep_dims=keep_dims)`